File: graphql-dgs-codegen-core/src/integTest/java/com/netflix/graphql/dgs/codegen/java/testcases/dataclasses/dataClassWithInterfaceInheritance/expected/DgsConstants.java
package com.netflix.graphql.dgs.codegen.java.testcases.dataclasses.dataClassWithInterfaceInheritance.expected;

import java.lang.String;

public class DgsConstants {
  public static final String QUERY_TYPE = "Query";

  public static class QUERY {
    public static final String TYPE_NAME = "Query";

    public static final String People = "people";
  }

  public static class TALENT {
    public static final String TYPE_NAME = "Talent";

    public static final String Firstname = "firstname";

    public static final String Lastname = "lastname";

    public static final String Company = "company";

    public static final String ImdbProfile = "imdbProfile";
  }

  public static class PERSON {
    public static final String TYPE_NAME = "Person";

    public static final String Firstname = "firstname";

    public static final String Lastname = "lastname";
  }

  public static class EMPLOYEE {
    public static final String TYPE_NAME = "Employee";

    public static final String Firstname = "firstname";

    public static final String Lastname = "lastname";

    public static final String Company = "company";
  }
}

File: graphql-dgs-codegen-core/src/test/resources/kotlin2/dataClassWithInterfaceInheritance/expected/DgsConstants.kt
package kotlin2.dataClassWithInterfaceInheritance.expected

import kotlin.String

public object DgsConstants {
  public const val QUERY_TYPE: String = "Query"

  public object QUERY {
    public const val TYPE_NAME: String = "Query"

    public const val People: String = "people"
  }

  public object TALENT {
    public const val TYPE_NAME: String = "Talent"

    public const val Firstname: String = "firstname"

    public const val Lastname: String = "lastname"

    public const val Company: String = "company"

    public const val ImdbProfile: String = "imdbProfile"
  }

  public object PERSON {
    public const val TYPE_NAME: String = "Person"

    public const val Firstname: String = "firstname"

    public const val Lastname: String = "lastname"
  }

  public object EMPLOYEE {
    public const val TYPE_NAME: String = "Employee"

    public const val Firstname: String = "firstname"

    public const val Lastname: String = "lastname"

    public const val Company: String = "company"
  }
}
